dotnet/Host.cs

```diff
@@ -0,0 +1,29 @@
+using System.Security.Cryptography;
+using Wasmtime;
+
+namespace WasmtimeDemo
+{
+    class Host : IHost
+    {
+        // These are from the current WASI proposal.
+        const int WASI_ENOTSUP = 58;
+        const int WASI_SUCCESS = 0;
```

Comment:
In the latest WASI ABI, these have names like `WASI_ERRNO_NOTSUP` and `WASI_ERRNO_SUCCESS` because they're now being autogenerated as "WASI\_\<errnoname>\_\<valuename>".

Reply:
I'll update to match.